docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/minilanding/Minilanding.scss

```diff
@@ -15,6 +15,11 @@
   }
 }
 
+div.row.feature-primary.flex-columns {
+  display: -webkit-box;
```

Comment:
Can probably do without this? Autoprefixer should take care of this.

Reply:
Checked, you're right it's redundant

Reply:
Looks good on the ODE:  http://ncigovcdode383.prod.acquia-sites.com/about-nci/organization/ccct